.gitignore

```diff
@@ -1,2 +1,3 @@
-target
-Cargo.lock
+/target
```

Comment:
My knowledge regarding gitignore is slightly sketchy, but I think
`target/` would match both `/target` and `/cargo-psp/target`

Reply:
i made it just `target`